src/rtctools/optimization/pi_mixin.py

```diff
@@ -290,3 +294,28 @@ def set_unit(self, variable: str, unit: str):
         ), "set_unit can only be called after read() in pre() has finished."
         self.__timeseries_import.set_unit(variable, unit, 0)
         self.__timeseries_export.set_unit(variable, unit, 0)
+
+
+def get_timeseries_from_pi(
+    file: Path, data_config=rtc.DataConfig, is_binary=False, validate=False
+) -> Tuple[List[datetime], Dict[str, np.ndarray]]:
```

Comment:
If this method is only used for testing, it should not be included here

Reply:
This method seems to have little added value over the current pi.Timeseries class, so I'll just remove it.